requirements.txt

```
@@ -1,2 +1,2 @@
-paho-mqtt==1.1
-robotframework==2.8.7
+paho-mqtt>=1.1
```

seems like the latest paho-mqtt is 1.4 and it worked without any issues:
```suggestion
paho-mqtt~=1.4
```

---

If you define version as `>=`, it will install the latest available version, but will be compatible with lower versions.
I've just tried reinstalling dependencies and got `paho_mqtt` version 1.4.0 and `robotframework` version 3.1.2.

I just didn't see why we should up the version requirement. I can replace it with `paho-mqtt~=1.1`, which would be a bit more restrictive and would not allow 2.x.

---

`>=` would use the latest version, as long as it is greater than or equal to specified version, including major version bumps that might not be backwards compatible. That's why I prefer `~=` so it stays within the major version 1.x. Either `~=1.1` or `~=1.4` is fine.
Same for robotframework, either `~=3.0` or `~=3.1` is fine.

https://www.python.org/dev/peps/pep-0440/#inclusive-ordered-comparison

---

Ok, done!